src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-logger",
-  "version": "0.2.9",
+  "version": "1.2.9",
```

Comment:
This should be 1.0.0. When we bump minor or major versions, the lesser numbers reset to 0.  Can you also update the package.json version at the root to match?

Thanks!